internal/shellgen/tmpl/flake_remove_nixpkgs.nix.tmpl

```diff
@@ -3,9 +3,14 @@
 
    inputs = {
      nixpkgs.url = "{{ .NixpkgsInfo.URL }}";
+
      {{- range .FlakeInputs }}
      {{.Name}}.url = "{{.URLWithCaching}}";
      {{- end }}
+
+     {{- if .PatchGlibc }}
+     nixpkgs-unstable.url = "nixpkgs";
```

Comment:
It should be locked by the flake.lock file. I'm not as familiar with the updating logic, but I think it should only change when `devbox update` is run.

Reply:
So, generally speaking, `devbox update` doesn't affect the `flake.lock` directly. It does affect how the `flake.nix`'s inputs are defined, and in that manner will affect the `flake.lock` file.

I think this could be problematic. My understanding is:
1. In general, in a flake, `nixpkgs` will resolve to the current `nix channel` for the user. 
2. The `flake.lock` will be generated from that. For users on a team, this could pin `nixpkgs` to different commit-hashes in the generated flake.lock, assuming they are even on the same nix channel.

Let me think some more (have to go to a meeting, jsut sharing my prelim. thoughts)